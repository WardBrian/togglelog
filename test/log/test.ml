let my_greeting () = "hi there" [@@warning "-32"]

let () =
  [%toggle_log "TESTING" (my_greeting ())];
  [%toggle_log "hi there (unlabled)"]
