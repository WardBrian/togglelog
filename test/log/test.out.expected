[LOG (TESTING) - test/log/test.ml:4:2] hi there
[LOG - test/log/test.ml:5:2] hi there (unlabled)
